### PR TITLE
[FEATURE]: Add categorical view to FAQ

### LINKED ISSUE #176: [FEATURE] Divide the FAQ for each category
### Detailed description

Divide the FAQ and put them in their category.

You will see the FAQ and categories here 👇 
https://docs.google.com/spreadsheets/d/1VyrRce4qvh2b6rNmbO4QmquQt0eN3_gsAoNhbxWU-jI/edit#gid=0

### Context

I think this makes the page clean, currently, the FAQ page is messed for me https://devprotocol.xyz/faq

### Possible implementation

Make something like this if possible 👉 https://rally.io/evergreen/faq 

or suggest  your ideas below

### Additional information

Thank you :)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Simperfy**
#### Description of the change
- FAQ now viewable in each category
- Updated the [CSV format](https://docs.google.com/spreadsheets/d/1VyrRce4qvh2b6rNmbO4QmquQt0eN3_gsAoNhbxWU-jI/edit#gid=0)
  - Note 1: only excludes the value in csv if all fields are omitted
  - Note 2: expects each language to have a pair question and answer
  - Note 3: expects to have the `categories` header column
  - Note 4: the question header column must follow this syntax `"Question (LANGUAGE_NAME)"`
  - Note 5: the answer header column must follow this syntax `"Answer (LANGUAGE_NAME*)"`
  - Note 6 *(optional)*: the question and answer header are designed to be easily reconfigured to accept different header format eg: `"Question --- LANGUAGE_NAME"` or `"Answer --- LANGUAGE_NAME"`
- Fixed overlapping animation in FAQ
- Updated the UI Layout of FAQ
- Updated the design of scrollbar across the site.
- Closes #176

#### Screenshots
Current Design *(light mode)*:
image

Proposed changes *(light mode)*:
image

Current Design *(dark mode)*:
image

Proposed changes *(dark mode)*:
image

Proposed changes *(mobile view)*:
image

#### Checklist

**I agree to the following :-**

*   Added description of the change
*   I've read the [contributing guidelines](https://github.com/web3community/devprotocol.xyz/blob/main/CONTRIBUTING.md)
*   Search previous suggestions before making a new PR, as yours may be a duplicate.
*   I acknowledge that all my contributions will be made under the project's license.

Notes (**optional**): 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/177"></a>

### CODE REVIEW

File: src/components/Faq/Faq.vue [hunk 1]
```diff
@@ -94,24 +122,109 @@ export default {
   name: "FAQ",
   data() {
     return {
-      questionAndAnswers: [],
-      selectedLanguage: "English",
+      categories: {
+        /*
```
Comment (Simperfy) [inline src/components/Faq/Faq.vue:126]: I kept this for documentation, let me know if I shall remove this.

File: src/components/Faq/Faq.vue [hunk 2]
```diff
@@ -94,24 +122,109 @@ export default {
...
+          ...
+        */
+      },
+      selectedCategory: "",
+      selectedLanguage: "english",
+      categoriesArr: [],
       languagesArr: [],
-      isDropdownActive: false
+      isDropdownActive: false,
+      /**
```
Comment (Simperfy) [inline src/components/Faq/Faq.vue:148]: This one also.

**Comment by Simperfy** [comment]
Oh wait I need to fix the mobile view.

**Comment by AbhinavMV** [comment]
This looks really good. ❤
Found an issue (maybe).
if i open one question in a tab it remains open for all the other tabs as well 
for eg - opened first question in For Everyone
image
First question is already open after selecting For Creators
image

**Comment by Simperfy** [comment]
> This looks really good. ❤ Found an issue (maybe). if i open one question in a tab it remains open for all the other tabs as well for eg - opened first question in For Everyone image First question is already open after selecting For Creators image

Good catch, I'm working on it.

File: src/components/Faq/Faq.vue (hunk 1 above)
Comment (Panquesito7) [reply src/components/Faq/Faq.vue:126]: Documentation is always good. Let's keep it. 🙂

File: src/components/Faq/lang.csv [hunk 3]
```diff
@@ -1,307 +1,75 @@
-English,Japanese
-"{
-  ""question"": ""Where can I find the Dev Protocol roadmap?"",
-  ""answer"": ""Best place to find the current roadmap is on our website: https://devprotocol.xyz"",
```
Comment (Panquesito7) [inline src/components/Faq/lang.csv:4]: Why was this deleted? 👀

**Review by vinzvinci — CHANGES_REQUESTED** [review]
Can we extend this, so we don't need to scroll? image

File: src/components/Faq/lang.csv (hunk 3 above)
Comment (Simperfy) [reply src/components/Faq/lang.csv:4]: This is because vinzvinci gave a new [csv format](https://docs.google.com/spreadsheets/d/1VyrRce4qvh2b6rNmbO4QmquQt0eN3_gsAoNhbxWU-jI/edit#gid=0), see #176 for more info.

**Comment by Simperfy** [comment]
> Can we extend this, so we don't need to scroll? image

Sure, I'll see what I can do. 👍🏻 I did try to keep things concise so it looks neat design-wise.

*I also notice that in your given example in #176 they're not fully extending the category, I believe this is related to design and they want to keep things concise. This is where I got my idea.*

image

**Comment by vinzvinci** [comment]
> > Can we extend this, so we don't need to scroll? image
> 
> Sure, I'll see what I can do. 👍🏻 I did try to keep things concise so it looks neat design-wise.
> 
> _I also notice that in your given example in #176 they're not fully extending the category, I believe this is related to design and they want to keep things concise. This is where I got my idea._
> 
> image

Heheh yeah sorry about that, I don't recommend scroll but if there's no more choice, let's choose that.

**Review by krishguptadev — APPROVED** [review]

**Comment by Simperfy** [comment]
I updated the category styles, it still looks good to me.

image

**Review by vinzvinci — APPROVED** [review]
It looks good because you are also good 😄 I approved new changes, I think there's still a request above. Will keep looking at this

**Comment by Simperfy** [comment]
> It looks good because you are also good smile I approved new changes, I think there's still a request above. Will keep looking at this

Glad to hear that ❤️. What request? Are you refering to @AbhinavMV's concern? I already fixed that one too.

File: src/styles/global.css [hunk 4]
```diff
@@ -167,3 +167,21 @@
...
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-thumb {
+  background: #888;
+  border-radius: 10px;
+}
+::-webkit-scrollbar-thumb:hover {
+  background: #555;
+}
```
Comment (Panquesito7) [inline src/styles/global.css:187]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by vinzvinci** [comment]
> 

Yep, I see and check now. Everything looks good to me

**Review by vinzvinci — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]
